### ISSUE TITLE
Add lang-attribute and make it change after language change

### ISSUE THREAD

**Issue by csckata** [BioDT/biodt-shiny#100]
The whole system is missing lang-attribute, which should be added at some point to tell the screenreaders (and better reach SEO-wise) what language the app is. Also, if we're going multilingual, the attribute should change according to the language used.

**Comment by csckata** [comment]
Lang-attribute has ben added, further development will be made if language change will be implemented
